calculator.js

```diff
@@ -0,0 +1,20 @@
+var calculator ={
+add:function (x,y) {
+  return x+y;
+}
+}
+  var calculator ={
+  subtract:function (x,y) {
+    return x-y;
+}
```

Comment:
Every time you write `var calculator =`, you're ovewriting/destroying the previous `var calculator`.